x-pack/platform/plugins/shared/fleet/server/tasks/unenroll_inactive_agents_task.ts

```diff
@@ -149,7 +153,7 @@ export class UnenrollInactiveAgentsTask {
         continue;
       }
       agentCounter += res.agents.length;
-      if (agentCounter >= UNENROLLMENT_BATCHSIZE) {
+      if (agentCounter >= this.unenrollBatchSize) {
```

Comment:
shouldn't this condition be `agentCounter > this.unenrollBatchSize`, so the unenroll goes ahead on equality?

Reply:
Yes, I wanted to check that in CI fails first. And then change it.

I forgot to create the PR as draft...